1/Dockerfile

```diff
@@ -49,6 +49,35 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
     chown -R 1001:0 /opt/openshift && \
+    chown -R 1001:0 /etc/alternatives && \
+    chown -R 1001:0 /var/lib/alternatives && \
+    chown -R 1001:0 /usr/lib/jvm && \
```

Comment:
what are we doing at assemble time (that's the only thing that runs under uid 1001) that needs ownership of these directories?